static/app/views/acceptOrganizationInvite/index.spec.tsx

```diff
@@ -245,11 +245,8 @@ describe('AcceptOrganizationInvite', function () {
     );
 
     expect(screen.getByTestId('existing-member')).toBeInTheDocument();
-
     await userEvent.click(screen.getByTestId('existing-member-link'));
-
-    expect(logout).toHaveBeenCalled();
-    await waitFor(() => expect(window.location.replace).toHaveBeenCalled());
```

Comment:
Removing this line because redirect was moved into `logout` function, which was mocked.